go/lib.h

```diff
@@ -96,3 +96,94 @@ void close_client(const void *client_ptr);
  * * The contained `connection_error_message` must not be null.
  */
 void free_connection_response(struct ConnectionResponse *connection_response_ptr);
+
+enum RequestType {
```

Comment:
Is it possible to move enum(s) to another file(s) and `#include` them?

On another hand we have permission to use cbindgen which can generate entire header automatically.